install-roles/cluster-role.yaml

```diff
@@ -5,58 +5,6 @@ kind: ClusterRole
 metadata:
```

Comment:
Can we add a comment at the top of this file that says which versions it is applicable to? 
```
These minimal values files are valid with fusion version:
  5.3.0
 .... etc
```

Reply:
Added comments to role and clusterroles files